vembrane/__init__.py

```diff
@@ -80,7 +82,7 @@ def filter_vcf(vcf: VariantFile, expression: str) -> Iterator[VariantRecord]:
             continue
```

Comment:
If `annotations == []` (which happens if there's no ANN INFO field in the header), the record will never get filtered, even if the expression evaluates to false